src/searchForm/autoComplete/section/index.tsx

```diff
@@ -6,37 +6,37 @@ import autoComplete from './AutoCompleteSection'
 const StyledAutoComplete = styled(autoComplete)`
   & {
     box-sizing: border-box;
-  }
-
-  & .kirk-textField-wrapper {
-    background: transparent;
-    border-radius: 0;
-    border: 0;
-  }
-
-  & .kirk-textField-wrapper input {
-    font-size: ${font.m.size}
-    line-height: ${font.m.lineHeight}
-    background: transparent;
-    border-radius: 0;
-  }
-
-  & .kirk-textField-wrapper .kirk-loader {
-    position: absolute;
-    top: 12px;
-    right: 12px;
-  }
-
-  & .kirk-items-list {
-    padding: 0;
-  }
-
-  & .kirk-items-list.kirk-autoComplete-list .kirk-items-list-item::before {
-    content: none;
-  }
 
-  & .kirk-items-list .kirk-items-list-item .kirk-item-choice {
-    padding: ${space.m} ${space.l};
+    .kirk-textField-wrapper {
+      background: transparent;
+      border-radius: 0;
+      border: 0;
+
+      input {
+        font-size: ${font.m.size}
+        line-height: ${font.m.lineHeight}
+        background: transparent;
+        border-radius: 0;
+      }
+
+      .kirk-loader {
+        position: absolute;
+        top: 12px;
+        right: 12px;
+      }
+    }
+
+    .kirk-items-list {
+      padding: 0;
+
+      &.kirk-autoComplete-list .kirk-items-list-item::before {
+        content: none;
+      }
+
+      .kirk-items-list-item .kirk-item-choice {
```

Comment:
`.kirk-items-list-item` has ben already declared.  On line **32**, do you need this strong selector or can you merge both of them?

Reply:
Same comment as above, I'll make the change, thanks ;)